src/common/directives/autoFocus.directive.js

```diff
@@ -3,11 +3,13 @@ import angular from 'angular';
 let directiveName = 'autoFocus';
 
 /* @ngInject */
-function desigSrc() {
+function desigSrc($timeout) {
   return {
     restrict: 'A',
     link:     function ( scope, element ) {
-      element[0].focus();
+      $timeout(function () {
```

Comment:
Without the timeout, it worked as intended on desktop, but on mobile it did a crazy scroll thing (see jira task).  Adding a timeout fixed it (I remembered that from ERT).  I will try a $digest and make another PR if that works.